src/Components/Books.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Book from './Book';
+import Form from './Form';
+
+function Books() {
+  const books = [{
+    id: 1,
+    name: 'ABC',
```

Comment:
- [ ] [OPTIONAL] It is better to change the `name` attribute to the `title` attribute as you used it in the `Book` component.